db/migrate/20191109022931_update_favorites_on_tracks.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateFavoritesOnTracks < ActiveRecord::Migration[6.0]
+  def change
+    asset_ids = Track.favorites.map(&:asset_id).uniq
```

Comment:
This will instantiate 45k records. I recommend using `pluck(:asset_id)` vs `map` to only select the field we need.